src/projects.md

```diff
@@ -3,255 +3,162 @@ title: Our Projects
 date: 2015-11-03T02:55:22-07:00
 author: elina
 layout: page
----
-
-OpenOakland projects are the core of our organization's efforts to connect citizens with impactful civic initiatives. **We're always seeking new projects and project leaders** to work on housing, transportation, education, health, or any area that can improve the lives of Oakland's residents. Please feel free to [join our Slack](https://docs.google.com/forms/d/e/1FAIpQLSee_qdE0qCmhufJC94MmSRVDLPAhhFJO4QMzuC31Kh0lxI_Mg/viewform?usp=sf_link) and reach out to members, leadership, or any project team for more information. You can contact project teams directly in their individual Slack channels. **Join or sign-in to Slack: <a href="http://slack.openoakland.org/" target="_blank" rel="noopener noreferrer">slack.openoakland.org</a>**
 
----
+active_projects:
+  -
+    name: Councilmatic
+    image: councilmatic_300x118.png
+    link: http://councilmatic.aws.openoakland.org/pc/
+    leader: Howard Mattis
+    slack_id: C0M89GTRT
+    slack_channel: councilmatic
+    description: We make the Oakland City Council meetings easily accessible to the citizens of Oakland. Using our site citizens can learn when are the upcoming city council meetings, view the agenda, put the meeting on your calendar, and send an electronic comment to the Council. Plus you can see videos of past meetings.
```

Comment:
I'm a still a little unfamiliar with data organization/storage in Jekyll so maybe @adborden can weigh in here - could all this `project` data be migrated to a collection or into a `_data` file/folder? Or is front matter the best way to store this? Since they're all on the same page (for now), we'd still be able to loop through each project in another directory and use the project layout that was created in this PR?

Reply:
This! 👆

Reply:
I don't know exactly how that works, either--what format can a data file take in the _data folder? But I would say we should _definitely_ do that. Two files: `active_projects.md` (or .yml or .json or whatever it needs), and `inactive_projects`. Makes moving a project form active -> inactive and vice versa so clear, simple. so good!

**Can I do that before we merge this PR? (Contingent upon figuring out what format the data can take)**

Reply:
`_data` files can be a variety of formats! https://jekyllrb.com/docs/datafiles/ Looks like keeping with the .yml standard might work well in this case. Love the idea for two separate files for active and inactive projects!

I think you could do that within this PR, as it'd mostly be transferring the already-typed content to a different location, and changing the source of the data that goes through the loops, i.e. `site.data.projects.active` or however you choose to nest/store the data. Check out `navigation.yml` and the corresponding line in `header.html`: `default_paths = site.data.navigation` for an example.

Reply:
Ok, that worked! But it raised another consideration: as far as I can figure, [data files can't use markdown syntax](https://jekyllrb.com/tutorials/orderofinterpretation/). I think we'd like our content contributors to be able to contribute their text content as markdown so they can avoid using tags like \<strong>, \<em>, and \<a>, and instead use * *, _ _, and \[ ]( )

I wonder if there's a Jekyll plugin or something that would "pre-interpret" our markdown data?

Reply:
got it! `markdownify` filter https://jekyllrb.com/docs/liquid/filters/